docs/index.bs

```diff
@@ -1965,17 +1961,14 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
         1. Run these substeps <a>in parallel</a>:
             1. Let |responseArray| be an empty array.
             1. If the optional argument |request| is omitted, then:
-                1. For each <a>fetching record</a> |entry| of its <a>request to response map</a>, in key insertion order:
-                    1. Add a  copy of |entry|.\[[value]] to |responseArray|.
+                1. [=list/For each=] |item| of the [=context object=]'s [=request response list=]:
+                    1. Add a copy of |item|'s |response| to |responseArray|.
```

Comment:
This could be part of an additional PR though, as this PR doesn't introduce this problem.

Reply:
I changed it to using the internal structs instead of the JS objects, so it seems fine. But as Fetch provides the [cloning](https://fetch.spec.whatwg.org/#concept-response-clone) algorithm, I'll look at it further whether using that'd make it more precise.